code/procs/antagonist_procs.dm

```diff
@@ -68,6 +68,7 @@
 				traitor_mob.put_in_hand_or_drop(H)
 		H.secure_classes["z"] = RADIOCL_SYNDICATE
 		H.set_secure_frequency("z",the_frequency)
+		H.protected_radio = 1 //maybe protects conspirator radio?
```

Comment:
maybe????

Reply:
fixed it! now definitively prevents eavesdropping from metaneural users on the conspirators' randomized freeuqnecy while also still letting metaneural users drop in on other security frequencies for the departments.